unit_tests/test_aggregator.py

```diff
@@ -52,31 +44,21 @@ def mock_start_websocket_funcs(mocker):
 
 @pytest.fixture(autouse=True)
 def mock_execute_batch_command_methods(mocker):
-    """
-    Fixture mocks methods and external functions and other methods used in
-    execute_batch_command.
-    """
+    """Mock methods and external functions and other methods used in execute_batch_command."""
     mocker.patch("gsy_e_sdk.aggregator.ClientCommandBuffer.buffer_length",
                  new_callable=PropertyMock, return_value=3)
     mocker.patch("gsy_e_sdk.aggregator.ClientCommandBuffer.execute_batch",
                  return_value=test_batch_command_dict)
     mocker.patch("gsy_e_sdk.aggregator.Aggregator._post_request",
                  return_value=["some_transaction_id", True])
-    mocker.patch("gsy_e_sdk.aggregator.AggregatorWebsocketMessageReceiver"
-                 + ".wait_for_command_response",
-                 return_value=test_response)
```

Comment:
Is the first modification after the function-docstring the one you meant earlier about reverting the patch in this fixture?

Reply:
@ERafaelMartinez yes